### ISSUE TITLE
2.11.2 patch release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3710]
- #3654
- #3655
- #3659 
- #3656
- #3670
- #3669
- #3663 
- #3650
- #3661
- #3668 
- #3672
- #3682
- #3687
- #3679 
- #3674
- #3684 
- #3701 
- #3709

## Pick list
```
git cherry-pick e631a54b47a341232e75edecce914a3d880a0d31
git cherry-pick 1fa0e3e14e6d964e90c699373345b85b86523267
git cherry-pick d014b3fc362bf98367a39419744fb2092d85df8c
git cherry-pick e6a21c6ae2380bd9c969f7a25f46cb6e281c0fe5
git cherry-pick 59b3089f16d53d54be8768d5b4ee59caf38b657e
git cherry-pick c3d6f5cde25cc12d9a02309c984ce3a090f80ea2
git cherry-pick 9b2901577bfeb795d2566f3473c514f8c88e31d6
git cherry-pick 01eee2cb54fa3090b68c57e9ddbb8d7ed0da8aca
git cherry-pick aaa455b4ab5feed59f52c3638de4b4ca6e7863ac
git cherry-pick fc7be55baff47b4bfd266fe00a5b32509c3bfecd
git cherry-pick 5fd6c7bf411bf91cc49bd8c035c36d6bb9ab7516
git cherry-pick 08e5361688b682b1bc94a5eaa16dde4ebbd771f7
git cherry-pick 317d3e7213dae8512469746d945556c967d13565
git cherry-pick 6b46824d511704d30a7d465c86e167bd7c55850b
git cherry-pick 37683ea0750bc846f765f59cc08b3f2e98069a9e
git cherry-pick 19e423029fb3f9a596bdf88690d97b46dca68d2a
git cherry-pick 258e1725f06a82426e1c8037aa336b77a44aff19
git cherry-pick a97fff1c9307a7b042093cfd7da09e5e6f5b49f8
```

**Comment by joeparsons** [comment]
## Release notes snippet

```
- Close #3693 Force az_paragraphs_full_width library to load in the head alone. (#3709)
- Closes #3700: Update drupal/core-recommended to 10.3.4. (#3701)
- Close #3683 use migmag_lookup instead of migrate_lookup (#3684)
- Closes #3702: Update drupal/seckit requirement from 2.0.1 to 2.0.3 (#3674)
- Update drupal/menu_link_attributes requirement from 1.4.0 to 1.5.0 (#3679)
- Re-add ghost block fix. (#3687)
- Update drupal/chosen requirement from 4.0.1 to 4.0.3 (#3682)
- Closes #3671 Add minified option on az_bootstrap_js_info (#3672)
- Closes #3666 Update Photo Gallery to prevent potential XSS vulnerability (#3668)
- Update drupal/media_library_edit requirement from 3.0.3 to 3.0.4 (#3661)
- Update drupal/antibot requirement from 2.0.3 to 2.0.4 (#3650)
- Update drupal/media_entity_file_replace requirement from 1.2.0 to 1.3.0 (#3663)
- Update drupal/flag requirement from 4.0-beta4 to 4.0.0-beta5 (#3669)
- Update drupal/media_library_form_element requirement from 2.1.0 to 2.1.1 (#3670)
- Update drupal/menu_link_attributes requirement from 1.3.0 to 1.4.0 (#3656)
- Close #3658 Add all paragraph migrations as optional dependencies of the news node migration. (#3659)
- Update drupal/bootstrap_utilities requirement from 3.0.0 to 3.0.1 (#3655)
- Closes #3637: Update drupal/intelligencebank requirement from 4.1.0 to 4.1.1 (#3654)
- Back to dev.
```

**Comment by joeparsons** [comment]
https://github.com/az-digital/az_quickstart/releases/tag/2.11.2
